2_observations.yml

```diff
@@ -96,8 +97,37 @@ targets:
     command: summarize_dat(in_ind = '2_observations/out/obs_flow_drb.rds.ind', out_file = target_name)
 
   # reservoir release data
+
   2_observations/out/reservoir_releases.csv.ind:
     command: clean_release_dat(
       in_ind = '2_observations/in/PCN_Spill_Release.xlsx.ind',
       out_ind = target_name,
       mgd_to_cms = I(0.0438125))
+
+  # reservoir diversion data
+  # you must download reservoir data from here: 'https://doimspp-my.sharepoint.com/:f:/g/personal/soliver_usgs_gov/EiawwIR59vdApLCkeyob4t0Bx86d2SoTic8ahCm6MH5wRA?e=KEk1zE'
+  # unzip downloaded folder to 2_observations/in
+  # such that the structure becomes
+  # 2_observations/in/reservoir_data
+      # WY2009 data/
+      # ...
+      # 01416900.WY2008.xls
+      # ...
+  2_observations/out/observed_daily_diversions.csv.ind:
+    command: read_munge_dailies(out_ind = target_name)
```

Comment:
Could be nice to "diversion" in the name of these functions (`read_munge_dailies`, `read_munge_monthlies`, `interpolate_to_daily`) so it's clear what they do when searching through functions